Bottom: 4375c54f688e0c854a82ab9a1c7140f6d6617d6b
Top:    ef12221b9b980010ecf33567429f0b17dee2a194
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 12:51:45 +0100

Refresh of revert-change-change-c-to-trea

---

diff --git a/src/change.c b/src/change.c
index 9787165..61cb335 100644
--- a/src/change.c
+++ b/src/change.c
@@ -839,7 +839,8 @@ static void *
 ChangeTextSize (LayerTypePtr Layer, TextTypePtr Text)
 {
   int value = (Absolute != 0 ? 0 : Text->Scale) +
-              (double)(Absolute != 0 ? Absolute : Delta) / (double)FONT_CAPHEIGHT * 100.;
+              (double)(Absolute != 0 ? Absolute : Delta)
+                / (double)FONT_CAPHEIGHT * 100.;
 
   if (TEST_FLAG (LOCKFLAG, Text))
     return (NULL);
@@ -915,7 +916,8 @@ static void *
 ChangeElementNameSize (ElementTypePtr Element)
 {
   int value = (Absolute != 0 ? 0 : DESCRIPTION_TEXT (Element).Scale) +
-              (Absolute != 0 ? Absolute : Delta) * 100 / FONT_CAPHEIGHT;
+              (double)(Absolute != 0 ? Absolute : Delta)
+                / (double)FONT_CAPHEIGHT * 100.;
 
   if (TEST_FLAG (LOCKFLAG, &Element->Name[0]))
     return (NULL);
